app/controllers/clear_cms/contents_controller.rb

```diff
@@ -126,6 +128,38 @@ def import
       redirect_to({:action=>:index}, notice: 'Content imported successfully.')
     end
 
+    def csv_importer
+    end
+
+    def csv_import
+      headers = CSV.read(params[:dump][:file].path, headers: true).headers
+      attributes_updated = headers & ClearCMS::Content.csv_allowable_attributes
+        CSV.foreach(params[:dump][:file].path, headers: true) do |row|
+          begin
+            product = ClearCMS::Content.find(row["id"])
+          rescue Exception => e
+            redirect_to({:action => "csv_importer"}, notice: "Error importing content. #{e}. Fix errors and try again.") and return
+          else
+            if attributes_updated.include?("_type")
+              product._type = row["_type"]
+            end
+
+            if attributes_updated.include?("footnote_number")
+              product.footnote_number = row["footnote_number"]
+            end
+
+            if attributes_updated.include?("categories")
+              product.categories = (product.categories + row["categories"].split(', ')).map {|category| category.downcase}.uniq
```

Comment:
Instead of assuming there will ALWAYS be a , and a space to split on, I would split on just the , and then make sure you are doing both strip and downcase to clean them up for both categories and tags

http://ruby-doc.org/core-2.2.0/String.html#method-i-strip